### PR TITLE
Updating versions and release notes for the 2019-03-01-hybrid profile

### PR DESCRIPTION
**Pull request by bganapa**
## Checklist

- [x] I have read the [_Submitting Changes_](https://github.com/Azure/azure-powershell/blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](https://github.com/Azure/azure-powershell/blob/master/CONTRIBUTING.md)
- [x] The title of the PR is clear and informative
- [x] The appropriate [change log has been updated](https://github.com/Azure/azure-powershell/blob/master/CONTRIBUTING.md#updating-the-change-log)
- [x] The PR does not introduce [breaking changes](https://github.com/Azure/azure-powershell/blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [ ] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [ ] the changes have gone through a [cmdlet design review](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (Microsoft internal partners only)
    - [x] the cmdlet markdown files were [generated using the `platyPS` module](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/help-generation.md)

### CODE REVIEW

File: src/StackAdmin/AzureRM/AzureRM.psd1 [hunk 1]
```diff
@@ -118,17 +118,12 @@
             # IconUri = ''
 
             # ReleaseNotes of this module
-            ReleaseNotes = '## 2019.01
+            ReleaseNotes = '## 2019.04
```
Comment (bganapa) [inline src/StackAdmin/AzureRM/AzureRM.psd1]: 2019.05

**Comment by bganapa** [comment]
https://azuresdkci.westus2.cloudapp.azure.com/view/PowerShell/job/ps-sign-esrp/123/ - ps-sign

**Comment by bganapa** [comment]
Update Change logs

**Comment by bganapa** [comment]
new ps-sign https://azuresdkci.westus2.cloudapp.azure.com/view/PowerShell/job/ps-sign-esrp/124/

**Comment by markcowl** [comment]
@bganapa Once the above sign build looks good to you, please comment and I will merge this and create a release candidate.

**Comment by bganapa** [comment]
another ps-sign - https://azuresdkci.westus2.cloudapp.azure.com/view/PowerShell/job/ps-sign-esrp/125/artifact/src/Stack/

**Pull request merged**